src/server/api/controllers/qr.controller.js

```diff
@@ -0,0 +1,22 @@
+const knex = require('../../config/db');
+
+const get = async (id) => {
+  try {
+    const instances = await knex('game_instances')
+      .select('game_instances.game_code as code')
+      .where({ id });
+
```

Comment:
Hey Masume, in line 7 you assign a value ID equal what?
if you want to assign it for example to equal game_instances ID then i would do it like this .where('game_instances.id', id);

Reply:
Hey Amer, it is taking Id value from swagger so I am following documentation  rules available in /home/masi/projects/fp-class12/src/server/api/controllers/modules.controller.js line 20